### PR TITLE
Feature/light and vob misc

### PR DESCRIPTION
**Pull request by JucanAndreiDaniel**
add data for every misc vob, light/lightpreset and additional data for light_maps (TODO left: add normals array for light_map)

### CODE REVIEW

File: include/phoenix/cffi/Mesh.h [hunk 1]
```diff
@@ -28,6 +28,21 @@ PXC_API uint32_t pxMshGetVertexCount(PxMesh const* msh);
...
+PXC_API uint8_t pxMshGetPolygonFlagGetIsPortal(PxMesh const* msh);
+PXC_API uint8_t pxMshGetPolygonFlagGetIsOccluder(PxMesh const* msh);
+PXC_API uint8_t pxMshGetPolygonFlagGetIsSector(PxMesh const* msh);
+PXC_API uint8_t pxMshGetPolygonFlagGetShouldRelight(PxMesh const* msh);
+PXC_API uint8_t pxMshGetPolygonFlagGetIsOutdoor(PxMesh const* msh);
+PXC_API uint8_t pxMshGetPolygonFlagGetIsGhostGccluder(PxMesh const* msh);
+PXC_API uint8_t pxMshGetPolygonFlagGetIsDynamicallyLit(PxMesh const* msh);
+PXC_API int16_t pxMshGetPolygonFlagGetSector_Index(PxMesh const* msh);
+PXC_API uint8_t pxMshGetPolygonFlagGetIsLod(PxMesh const* msh);
+PXC_API uint8_t pxMshGetPolygonFlagGetNormalAxis(PxMesh const* msh);
```
Comment (lmichaelis) [inline include/phoenix/cffi/Mesh.h]: These signatures don't match the ones in the source file (-> `uint32_t i` is missing)

**Review by lmichaelis — COMMENTED** [review]
Thank you! Just one issue with the signature of the new mesh functions. LGTM otherwise :)

File: include/phoenix/cffi/Mesh.h (hunk 1 above)
Comment (JucanAndreiDaniel) [reply include/phoenix/cffi/Mesh.h]: oh, totally forgot about these, nice find

**Comment by lmichaelis** [comment]
Okay, I'll merge this but in the future, please split out unrelated changed in separate subsystems into multiple PRs :)

**Pull request merged**